scripts/run-bundled-codegen.sh

```diff
@@ -0,0 +1,64 @@
+# Bail if errors occur
+set -e
+
+# Get the path to the script directory
+SCRIPT_DIR="$(eval dirname $0)"
```

Comment:
Why is `eval` is necessary here?  The double quoted string would already interpolate variables.

Reply:
I feel like I was running into problems without it, but that may have been on an older version, lemme test without

Reply:
removed.